java/client/src/test/java/glide/api/RedisClientTest.java

```diff
@@ -5257,4 +5265,54 @@ public void blmove_returns_success() {
         assertEquals(testResponse, response);
         assertEquals(value, payload);
     }
+
+    @SneakyThrows
+    @Test
+    public void bitfieldReadOnly_returns_success() {
+        // setup
+        String key = "testKey";
+        Long[] result = new Long[] {7L};
+        BitFieldGet subcommand = new BitFieldGet(new SignedEncoding(4), new Offset(2));
+        String[] args = ArrayUtils.addFirst(subcommand.toArgs(), key);
```

Comment:
you aren't really testing subcommand.toArgs(), but just comparing `subcommand.toArgs() == subcommand.toArgs()` (which you know will be true).  
Instead, create the array yourself.